hpx/runtime/threads/policies/hwloc_topology_info.hpp

```diff
@@ -41,6 +42,35 @@ namespace hpx { namespace resource
 
 namespace hpx { namespace threads
 {
+    struct hpx_hwloc_bitmap_wrapper {
+        // take ownership of the hwloc allocated bitmap
+        hpx_hwloc_bitmap_wrapper(void *bmp) {
+            bmp_ = reinterpret_cast<hwloc_bitmap_t >(bmp);
+        }
+        // frees the hwloc allocated bitmap
+        ~hpx_hwloc_bitmap_wrapper() {
+            hwloc_bitmap_free(bmp_);
+        }
+        // stringify the bitmp using hwloc
+        friend HPX_EXPORT std::ostream& operator<<(std::ostream& os,
+            hpx_hwloc_bitmap_wrapper const* bmp);
+
+        // the raw bitmap object
+        hwloc_bitmap_t bmp_;
+    };
+
+    /// \brief Please see hwloc documentation for the corresponding
+    /// enums HWLOC_MEMBIND_XXX
+    enum hpx_hwloc_membind_policy : int {
+        HPX_MEMBIND_DEFAULT = 0,
```

Comment:
Please use the original hwloc constants to initialize the values of this enumerator. I also think that there is no need to prefix the enumerator value names with `HPX_`. Lastly, we usually use `ALL_CAPTIAL_NAMES` for macros only.